src/components/ReportActionItem/TaskView.js

```diff
@@ -69,7 +71,7 @@ function TaskView(props) {
                         accessibilityLabel={taskTitle || props.translate('task.task')}
                     >
                         {({pressed}) => (
-                            <>
+                            <OfflineWithFeedback pendingAction={lodashGet(props, 'report.pendingFields.reportName', null)}>
```

Comment:
Let's add pendingFields to the reportPropTypes, like this
https://github.com/Expensify/App/blob/f8bb2b116439ab5210c80eb807a1b843ef37307d/src/pages/settings/Profile/Contacts/NewContactMethodPage.js#L42-L43

Reply:
Also I don't think we need to pass `null` as the default value, since `undefined` is the default and should trigger the default prop to kick in.

Reply:
Ah yes pendingFields is an object of strings. My bad for missing that. The `errorFields` structure is correct; it's different from `pendingFields`, which is admittedly a bit confusing.